combat/attack.cpp

```diff
@@ -1,1327 +1 @@
-/*
```

Comment:
You fucked up the line endings here, make sure you're using the unix compatible crlf or whatever the git setting is.